apps/pxweb2/src/app/app.tsx

```diff
@@ -544,42 +545,45 @@ export function App() {
               {selectedNavigationView === 'save' && drawerSave}
               {selectedNavigationView === 'help' && drawerHelp}
             </NavigationDrawer>
+        </div>
           )}
         </div>
-        <div className={styles.mobileNavigation}>
+        <div className={`${styles.mobileNavigation} ${styles.scrollable}` }>
```

Comment:
We should be consistent and use clsx to merge several styles in a className.

Reply:
File now uses clsx to merge several styles